wiki/Music_theory/Time_signature/de.md

```diff
@@ -0,0 +1,14 @@
+---
+stub: true
+tags:
+  - sheet music
+  - Notenblatt
+---
+
+# Taktart
+
+*Für mehr Informationen über Taktarten in der Musiktheorie, siehe: [Taktangabe](https://de.wikipedia.org/wiki/Taktangabe)*
+
+Eine **Taktart** ist ein musikalischer Begriff, der die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) eines Songs (die obere Zahl), und welche Note einen einzelnen Beat markiert (die untere Zahl), definiert. Bei einem 4/4-Takt passen 4 1/4-Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Walzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4-Noten in einen Takt passen.
```

Comment:
Sorry, but re-reading this, the sentence does feel a bit like spaghetti. How about leaving out the `oben/unten` bit? It should become clear what the numbers mean when reading the `4/4` and `3/4` examples.

`Notenwert` is the technical term for the "length" of a `Note`. This might be more clear too?

This concept is a bit difficult to explain in a single sentence anyways, but I tried to make it a bit clearer.

```suggestion
Eine **Taktart** ist ein musikalischer Begriff, der die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) eines Songs und den Notenwert eines einzelnen Beats definiert. Bei einem 4/4-Takt passen 4 1/4-Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Walzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4-Noten in einen Takt passen.
```

Reply:
I do agree that the sentence is difficult to read. Music theory is hard to grasp in general and if someone wants more information about these terms, I think it is okay to let them do research on their own.

Looking at this and the proposal I made in #10176, I'm not sure about translating measure with `Takt` here. I do think it's okay in this context, though I wanted to mention it explicitly.

Reply:
I think `Takt` is a perfectly good translation for `measure` ([explanation](https://github.com/ppy/osu-wiki/pull/10176#issuecomment-1763375727)).

I think I have a better suggestion than my first one:

```suggestion
Die **Taktart** eines Songs definiert die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) (die obere Zahl) und welche Note einen einzelnen Beat markiert (die untere Zahl). Bei einem 4/4-Takt passen 4 1/4-Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Walzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4-Noten in einen Takt passen.
```

Let me know what you think

Reply:
```suggestion
Die **Taktart** eines Songs definiert die Anzahl an [Beats](/wiki/Music_theory/Beat) in einem einzigen [Takt](/wiki/Music_theory/Measure) (die obere Zahl) und den Notenwert eines einzelnen Beats (die untere Zahl). Bei einem 4/4-Takt passen 4 1/4-Noten (*Viertelnoten*) in einen Takt, wohingegen ein 3/4-Takt (auch bekannt als "Walzer" oder "Dreivierteltakt") bedeutet, dass 3 1/4-Noten in einen Takt passen.
```
How about mixing both?

Reply:
LGTM.